src/java.base/unix/native/libjava/UnixFileSystem_md.c

```diff
@@ -51,23 +51,8 @@
 #include "java_io_FileSystem.h"
 #include "java_io_UnixFileSystem.h"
 
-#if defined(_AIX)
-  #if !defined(NAME_MAX)
-    #define NAME_MAX MAXNAMLEN
-  #endif
-  #define DIR DIR64
-  #define dirent dirent64
-  #define opendir opendir64
-  #define readdir readdir64
-  #define closedir closedir64
-  #define stat stat64
-#endif
```

Comment:
same as above

Reply:
https://www.ibm.com/docs/en/aix/7.1?topic=volumes-writing-programs-that-access-large-files implies _some_ (like `stat`) are redirected by the macro, just not all (e.g. `readdir` isn't), I think?

Reply:
Indeed; the stat->stat64 conversion is handled; but the opendir etc remaps need to stay.